### DISCUSSION TITLE
Want help with getting feed to work

### DISCUSSION THREAD

**Discussion by cx48** [timlrx/tailwind-nextjs-starter-blog#1009] [Q&A]
Hello, I'm facing issue with getting comments section and feed to work.

For RSS, I'm using **/feed.xm**l after the domain where my blog is being hosted. I'm using RSS app to test if it works, for some reason it's failing to fetch any of my recent blogs

Also, can giscus comments be disabled on individual blogs and what about removing **Discuss on twitter** below every blog?

**Comment by FergusHuang** [comment]
Deleting comments  property from the siteMetadata variable in the data/siteMetadata.js file can disable the comment feature.
